<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New User</string>
  </property>
  <widget class="QLabel" name="userImage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>280</width>
     <height>190</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Quicksand</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Họ và tên:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>40</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Quicksand</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Giới tính:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>80</y>
     <width>350</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Quicksand</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ngày sinh:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="nameValue">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>250</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Quicksand</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>640</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Quicksand</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Lưu</string>
   </property>
  </widget>
  <widget class="QLabel" name="dayDob">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>120</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Quicksand</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ngày:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="dayDobValue">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>120</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Quicksand</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QTextEdit" name="monthDobValue">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>120</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Quicksand</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QLabel" name="monthDob">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>120</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Quicksand</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tháng:</string>
   </property>
  </widget>
  <widget class="QLabel" name="yearDob">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>120</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Quicksand</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Năm:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="yearDobValue">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>120</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Quicksand</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QLabel" name="status">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>160</y>
     <width>350</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Quicksand</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="male">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>40</y>
     <width>250</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Quicksand</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tích nếu bạn là nam</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
